.pre-commit-config.yaml

```diff
@@ -0,0 +1,11 @@
+default_stages: [commit]
+repos:
+- repo: local
+  hooks:
+
+    - id: version-check
+      name: Check if version(s) are in sync
+      always_run: true
+      verbose: true
+      language: system
+      entry: python scripts/version_check.py sqlalchemy_exasol/version.py
```

Comment:
Theoretically, you could simply use the nox command check_version

Reply:
yes, but for now I wanted nox to check and for the hook I wanted it to fix it if necessary.